README.md

```diff
@@ -159,6 +159,17 @@ OpenFunction is sponsored and open-sourced by the [KubeSphere](http://kubesphere
 
 - Slack: [#sig-serverless](https://kubesphere.slack.com/archives/C021XAR3CG3)
 - Wechat: join the OpenFunction user group by following the KubeSphere WeChat subscription
+
+## Landscapes
+
+<p align="center">
+<br/><br/>
+<img src="https://landscape.cncf.io/images/left-logo.svg" width="150"/>&nbsp;&nbsp;<img src="https://landscape.cncf.io/images/right-logo.svg" width="200"/>&nbsp;&nbsp;
+<br/><br/>
+OpenFunction enriches the <a href="https://landscape.cncf.io/serverless?license=apache-license-2-0">CNCF CLOUD NATIVE Landscape.
```

Comment:
Looks like `CNCF CLOUD NATIVE Landscape` can be changed to `CNCF Cloud Native Landscape`